src/gui/qgsvscrollarea.h

```diff
@@ -0,0 +1,44 @@
+/***************************************************************************
+                              qgsvscrollarea.h
+                              ------------------------
+  begin                : September 2017
+  copyright            : (C) 2017 Sandro Mani
+  email                : manisandro at gmail dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSVSCROLLAREA_H
+#define QGSVSCROLLAREA_H
+
+#include "qgis_gui.h"
+#include <QScrollArea>
+
+/**
+ * \ingroup gui
+ * QgsVScrollArea is a QScrollArea subclass which only displays a vertical
+ * scrollbar and fits the width to the contents.
+ *
+ * \since QGIS 3.0
+ */
+class GUI_EXPORT QgsVScrollArea : public QScrollArea
```

Comment:
@manisandro I think this should inherit from QgsScrollArea

Reply:
Ah yes, done in bb21395, thanks.